src/app/cake/models/cake.model.ts

```diff
@@ -0,0 +1,48 @@
+import Bread from '../../bread/bread.model';
+
+class Cake implements Bread {
```

Comment:
Why the cake class is implementing the Bread interface?

Reply:
I was just playing around with the inheritance stuff, since a cake is a kind of bread or is composed by a bread...I forgot to remove that test. I'll get rid of it on next deliverable.